doc/nrf/app_approtect.rst

```diff
@@ -0,0 +1,110 @@
+.. _app_approtect:
+
+Enabling access port protection mechanism
+#########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. app_approtect_info_start
+
+Several Nordic Semiconductor SoCs supported in the |NCS| offer an implementation of the access port protection mechanism (APPROTECT).
+When enabled, this mechanism blocks the debugger from read and write access to all CPU registers and memory-mapped addresses.
+Accessing these registers and addresses again requires disabling the mechanism and erasing the flash.
+In factory state, all devices come with the access port protection enabled.
```

Comment:
Not correct, only devices with hardened / improved approtect come factory enabled. Nrf91 and older nrf52s will have ap-protect disabled from the factory.